ouroboros-consensus-protocol/CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 # Changelog entries
 
+<a id='changelog-0.12.0.0'></a>
+## 0.12.0.0 -- 2025-04-21
+
+### Breaking
+
+- Restrict `ouroboros-consensus` to UTxO-HD versions (>=0.25).
```

Comment:
The cabal file itself mentions `ouroboros-consensus >=0.23 && <0.27,` so maybe something went wrong here unless I'm mistaken?